src/Symfony/Component/Security/Tests/Http/RememberMe/TokenBasedRememberMeServicesTest.php

```diff
@@ -138,6 +143,16 @@ public function testAutoLogin()
         $this->assertEquals('fookey', $returnedToken->getKey());
     }
 
+    public function provideUsernamesForAutoLogin()
+    {
+        return array(
+            // simple case
```

Comment:
When using a data provider, it's nice to put such comment as the last item of the values array. That way the message is displayed when the test fails.